src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
+    let recipeToRemove = this.recipesToCook.indexOf(recipe);
```

Comment:
might be good to call this variable something like `indexOfRecipeToRemove` to make it clear that it contains a number rather than an instance of the Recipe class, unlike the variable `recipe`

Reply:
This was changed this to reflect your suggestion.